hasura/graphql-engine
# fix sending duplicate content-type header to remote schemas  (fix #2159)

### Description


In remote schema when "forward all headers from client" option was checked, and the client sent a `content-type` header, there were two `content-type` headers that were being sent. Fixed that.

### Affected components 


- Server

### Related Issues



#2159 
### Solution and Design


Ignore any `content-type` header sent by the client.

### Steps to test and verify


1. Add a remote schema, with "forward client headers" option checked
2. Send a query to remote server with `content-type` header set
3. Observe, that `content-type` header is `application/json` and no duplicate headers are sent.
### Limitations, known bugs & workarounds
